ci/check_dependencies/helm_charts.py

```diff
@@ -43,9 +44,13 @@ def get_info() -> list[str]:
         pd_entries["created date"] = pd.to_datetime(
             pd_entries["created"], format=time_formate_with_zone
         )
-
-        pd_sorted_entries = pd_entries.sort_values("created date", ascending=False)
-        latest_release = pd_sorted_entries.loc[0].values
+        if dep["name"] in DEPENDENCIES_ONLY_MINOR_UPGRADES:
+            major_version, *_ = current_version.split(".")
+            compatible_entries = pd_entries[pd_entries["version"].str.startswith(major_version)]
```

Comment:
It would be nice to have unit tests for the tool :D
I have no idea how `pd_entries` structure looks like 😓